macros/src/from_macro.rs

```diff
@@ -558,7 +593,8 @@ pub fn implement_device(item: TokenStream) -> TokenStream {
                 .cloned()
                 .map(|f| device_driver_generation::Field {
                     name: f.name.to_string(),
-                    description: f.description,
+                    description: None,
+                    attributes: f.attributes,
```

Comment:
I really do want this description to be filled and have the attributes be purely `cfg`.
This probably works because it forwards the attributes including the doc attrs, but I don't think it's proper.

Reply:
My thinking for these is that the attributes are applied to a single item in the macro expansion, so there's no reason not to allow the user to provide whatever attributes they want to that item (for example maybe they want to `#[allow(non_camel_case)]` or `#[derive(Hash)]`).

The Register has to apply the attributes to multiple items so it made sense to me to restrict it to just the cfg and doc attributes which make sense for each of those items.